SinaiStCatherine/SinaiNewEt001.xml

```diff
@@ -72,28 +74,42 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </supportDesc>
-                </objectDesc>  
-                
-                
+                </objectDesc> 
                 <bindingDesc>
-                   <binding xml:id="binding">
-                    <decoNote xml:id="b1" type="Other">Non-Ethiopian binding, strongly damaged. Two leather boards, non-Ethiopian
-                    endboards, pastedowns made of parchment leaves with writing in Greek.</decoNote>
-                     <decoNote xml:id="b2" type="SewingStations">3</decoNote>
-                     <decoNote xml:id="b3" type="SewingStations">Sewing on three sewing stations, non-original; orginally there
-                     were two pairs.</decoNote>       </binding>
+                  <decoNote xml:id="b1" type="Other">Non-Ethiopian binding, originally probably manufactured for a different manuscript.  
+                    Wrapping made of stiff untooled leather embracing the textblock; spine lining. Turn-ins on the left side are crudely cut, of irregular shape.</decoNote>
+                  <decoNote xml:id="b2" type="Endbands">Endbands are made of threads, slightly protrude beyond the edges of the spine.</decoNote>
+                  <decoNote xml:id="b3" type="EndLeaves">The pastedown on the left side is placed under the turn-ins, 
+                    the pastedown on the right side is placed over the turn-ins; both made of parchment leaves with writing in Greek, originating probably from two different manuscripts.</decoNote>                      
+                  <decoNote xml:id="b4" type="SewingStations">3</decoNote>
+                  <decoNote xml:id="b5" type="SewingStations">Sewing on three sewing stations, probably non-original; orginally there
+                    were two pairs (see, e.g., the centre fold between <locus target="#13v"/> and <locus target="#14r"/>).</decoNote>
+                  <decoNote xml:id="b6" type="bindingMaterial">
+                    <material key="leather"/>
+                    <material key="textile"/>
+                  </decoNote>
                 </bindingDesc>
-              </physDesc>               
+                </physDesc>               
               
-           <!--   <history>
-                
-              </history>-->
+           <history>
+             <origin>
+               <origDate notBefore="1320" notAfter="1420" evidence="lettering">The manuscript is composed of three units. 
+                 <ref target="#p1">Unit 1</ref> and <ref target="#p3">Unit 3</ref> can be dated to <date>the 14th century, probably around its middle</date>; 
+                 <ref target="#p2">Unit 2</ref>can be dated to <date>the second half of the 13th-/early 14th-century</date></origDate>.
+               <ref target="#p1">Unit 1</ref> belonged to the same manuscript as <ref target="#p3">Unit 3</ref> which was copied on paper outside Ethiopia, and did not survive.
```

Comment:
"did not survive in its entirety", "in its original unit of circulation"? "did not survive" by itself can be a bit misleading, because if it wouldn't have survived at all, it could not be described here, I think.

Reply:
ok